tests/test_all.sh

```diff
@@ -10,32 +10,42 @@
 # # # for more details. If you did not receive the license, for more information see:
 # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-# Please identify and update all instances of "@" found in this script appropriately.
-# You will need to generate one or more test scripts that test your complete functionality,
-# (these scripts provide example geoips calls and sample output, as well as providing a full integration test),
-# and call each script within the test_all.sh script.  Do not rename this script or test directory - automated
-# integrations tests look for the tests/test_all.sh script for complete testing.
+# @ Please identify and update all instances of "@" found in this script appropriately.
+# @ You will need to generate one or more test scripts that test your
+# @ complete functionality, @(these scripts provide example geoips calls and
+# @ sample output, as well as providing a full integration test),
+# @ and call each script within the test_all.sh script.
+
+# Do not rename this script or test directory - automated integration
+# tests look for the tests/test_all.sh script for complete testing.
 
 #!/bin/bash
 
-# This should contain test calls to cover ALL required functionality tests for the @package@ repo.
+# This should contain test calls to cover ALL required functionality tests
+# for the @package@ repo.
 
 # The $GEOIPS tests modules sourced within this script handle:
-   # setting up the appropriate associative arrays for tracking the overall return value,
+   # setting up the appropriate associative arrays for tracking the overall
+   #   return value,
    # calling the test scripts appropriately, and 
    # setting the final return value.
 
 # Note you must use the variable "call" in the for the loop
 
 . $GEOIPS/tests/utils/test_all_pre.sh @package@
 
+# @ NOTE: Update "template_basic_plugin" paths below to point to your package's
+# @ test scripts, ie
+# @   $GEOIPS_PACKAGES_DIR/template_basic_plugin -> $GEOIPS_PACKAGES_DIR/@package@
+
 echo ""
 # "call" used in test_all_run.sh
 for call in \
 \
             "$GEOIPS_PACKAGES_DIR/geoips/tests/utils/check_code.sh all `dirname $0`/../" \
-            "$GEOIPS_PACKAGES_DIR/@package@/tests/scripts/test_config.sh" \
-            "$GEOIPS_PACKAGES_DIR/@package@/tests/scripts/@mydatatype@.tc.@product@.imagery_clean.sh"
+            "test_interfaces" \
+            "$GEOIPS_PACKAGES_DIR/template_basic_plugin/tests/scripts/test_config.sh" \
```

Comment:
@cpcamacho @jsolbrig @evrose54 Planning to make these actually installable and runnable - so we can actually make sure our template is properly formatted and functional.  Will make these changes directly on this branch.